lib/r10k/cli/puppetfile.rb

```diff
@@ -46,7 +46,7 @@ def self.command
           usage 'check'
           summary 'Try and load the Puppetfile to verify the syntax is correct.'
 
-          required nil, :puppetfile, 'Path to Puppetfile (can also be set with PUPPETFILE environment variable)'
```

Comment:
Should we actually remove these in the deprecation PR, that way we're not pointing users at them?